src/core/Akka.Streams/Implementation/Stages/Stages.cs

```diff
@@ -479,7 +479,9 @@ public SymbolicGraphStage(ISymbolicStage<TIn, TOut> symbolicStage) : base(symbol
     /// </summary>
     /// <typeparam name="TIn">TBD</typeparam>
     /// <typeparam name="TOut">TBD</typeparam>
+#pragma warning disable CS0618 // Type or member is obsolete
     public interface ISymbolicStage<in TIn, out TOut> : IStage<TIn, TOut>
```

Comment:
Is it `IStage` that is obsolete? Any reason why it was marked as Obsolete (in the `Reason` field?)

Reply:
@eaba so let's use `GraphStage` instead of `IStage`, rather than disable the warning.

Reply:
Ah, I think I see what the issue is here - this is an interface not a class - therefore it can't be fixed via inheriting from GraphStage, got it.